### PR TITLE
feat(crons): Add test for basic monitor ingestion

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This is extracted from https://github.com/getsentry/relay/pull/2448

It was actually incorrectly named in that PR and pretty unrealted to the changes

#skip-changelog

### CODE REVIEW

File: tests/integration/test_monitors.py [hunk 1]
```diff
@@ -10,6 +11,26 @@ def generate_check_in(slug):
     }
 
 
+def test_monitor_ingest(mini_sentry, relay):
```
Comment (iker-barriocanal) [inline tests/integration/test_monitors.py:14]: Not sure if I understand. What's the behavior we're trying to test? Looking at the test below the only difference I see is the `monitors_consumer` fixture, but not relay behavior. Maybe I'm missing something?

Comment (evanpurkhiser) [reply tests/integration/test_monitors.py:14]: This test doesn’t run the full processing pipeline. I believe it’s faster cc @jan-auer

**Review by jan-auer — APPROVED** [review]

File: tests/integration/test_monitors.py (hunk 1 above)
Comment (jan-auer) [reply tests/integration/test_monitors.py:14]: That's right. This test runs without processing mode and doesn't need Kafka, so it's a fair bit faster to run. Also, it ensures that PoPs work correctly in forwarding their data.

**Comment by jan-auer** [comment]
@evanpurkhiser please run `make format-python`; then this is good to merge!

**Pull request merged**